log.js

```diff
@@ -13,57 +13,36 @@ winston.addColors({
   debug: 'grey',
 })
 
-logger.clear()
-if (process.env.NODE_ENV === 'test') {
-  logger.add(winston.transports.File, {
-    name: 'log',
-    level: 'debug',
-    filename: 'tests.log',
-    json: false,
-    colorize: true,
-    prettyPrint: true,
-  })
-} else if (process.env.NODE_ENV === 'production') {
-  logger.add(winston.transports.Console, {
-    name: 'log',
-    level: 'info',
-    json: true,
-    colorize: false,
-    prettyPrint: false,
-    silent: false,
-    timestamp: true,
-    stringify: true,
-    handleExceptions: true,
+const flattened = winston.format((meta) => {
+  const { message, level, timestamp: ts, ...rest } = meta
+  const result = flattenMeta(rest)
+  Object.entries(result).forEach(([key, val]) => {
+    // eslint-disable-next-line no-param-reassign
+    meta[key] = val
   })
```

Comment:
this is really weird - for some reason you return anything that isn't the original meta object or it breaks logging entirely. Even if I create an object with the exact same state..